meteor-app/imports/ui/Viewers_List/Viewers_List.tsx

```diff
@@ -15,11 +17,39 @@ const usePage = (account_id) => useTracker(() => {
 
 export const Viewers_List = () => {
 
-	const { account_id } = useContext(UserContext)
+	const { user, account_id } = useContext(UserContext)
 	const { isLoading, viewers } = usePage(account_id)
+	const [query, setQuery] = useState('')
+	const [viewersShown, setViewersShown] = useState(viewers)
+	let viewersShownRegex = new RegExp(query, "i")
+
+	if (!user) {
+		return <Redirect to='/login' />
+	}
+
+	useEffect(() => {
+		filterDishes()
+	}, [query])
+
+	useEffect(() => {
+		setViewersShown(viewers)
+	}, [viewers])
+
+	const filterDishes = () => {
```

Comment:
What is "Dishes" ?

Reply:
Oh I found that function somewhere and modified it, forgot to change the function name